src/main/resources/swagger.api/schemas/parameter.json

```diff
@@ -1,13 +1,13 @@
 {
   "description": "List of key/value parameters of an error",
   "type": "object",
-  "properties": {
+  "parameters": {
```

Comment:
It was just each "description" that needed to change. Not the actual JSON property.